webapp/src/components/checklist_item/checklist_item.tsx

```diff
@@ -61,6 +61,12 @@ export const ChecklistItem = (props: ChecklistItemProps): React.ReactElement =>
 
     const toggleDescription = () => setShowDescription(!showDescription);
 
+    useEffect(() => {
```

Comment:
Even though this doesn't cause double-renders, `useUpdateEffect` might be a bit more in line with the usage here. (0/5)

```suggestion
    useUpdateEffect(() => {
```


Also just a quick plug for `useProxyState` that was written with this pattern in mind, but also 0/5 on this!

Reply:
Thanks! useUpdateEffect is what I was looking for.